ParseSwift.playground/Pages/15 - Custom ObjectId.xcplaygroundpage/Contents.swift

```diff
@@ -21,7 +21,7 @@ npm start -- --appId applicationId --clientKey clientKey --masterKey masterKey -
 initializeParseCustomObjectId()
 
 //: Create your own value typed `ParseObject`.
-struct GameScore: ParseObject {
```

Comment:
change this to `struct GameScore: ParseObject, ParseObjectMutable`, though the line you have will work it shows that if developers want to make their object mutable, they can by adding/removing the second protocol